Data/market-hours/market-hours-database.json

```diff
@@ -5789,7 +5873,8 @@
         "1/30/2017",
         "2/16/2018",
         "2/5/2019",
-        "1/27/2020"
+        "1/27/2020",
+        "14/04/2017"
```

Comment:
No worries, maybe in the future we'll change it to a more international standard format, such as  [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) (yyyy-MM-dd)